tracardi/process_engine/action/v1/contains_string_action.py

```diff
@@ -0,0 +1,102 @@
+from pydantic import validator
+
+from tracardi.service.plugin.domain.config import PluginConfig
+
+from tracardi.service.plugin.domain.register import Plugin, Spec, MetaData, Documentation, PortDoc, Form, FormGroup, \
+    FormField, FormComponent
+from tracardi.service.plugin.runner import ActionRunner
+from tracardi.service.plugin.domain.result import Result
+
+
+class Config(PluginConfig):
+    field: str
+    prefix: str
+
+    @validator("prefix")
+    def if_prefix_is_empty(cls, value):
+        if value == "":
+            raise ValueError("Prefix cannot be empty")
+        return value
+
+
+def validate(config: dict) -> Config:
+    return Config(**config)
+
+
+class ContainsStringAction(ActionRunner):
+
+    def __init__(self, **kwargs):
+        self.config = validate(kwargs)
+
+    async def run(self, payload: dict, in_edge=None):
+        dot = self._get_dot_accessor(payload)
+        if self.config.prefix in dot[self.config.field]:
```

Comment:
error prone, you should first check if dot[self.config.field] is iterable, and log, then throw an exception with an explanation